train_test.py

```diff
@@ -0,0 +1,85 @@
+import math
+import unittest
+
+import torch
+
+from train import scribe_loss
+
+
+class TestTrain(unittest.TestCase):
+
+    def test_scribe_loss_penup_one_matches(self):
```

Comment:
Nice job with the tests! I didn't review in so much detail but it seems like you were quite comprehensive.